programs/server/config.xml

```diff
@@ -45,6 +45,14 @@
         -->
     </logger>
 
+    <send_crash_reports>
```

Comment:
It's Ok to mention Sentry here, so that people will be aware that the data is send to 3rd party service (not to our own servers).

Reply:
ok